Templates/Tests/Expected/AutoMockable.expected

```diff
@@ -79,41 +88,52 @@ class ExtendableProtocolMock: ExtendableProtocol {
         return reportMessageCallsCount > 0
     }
     var reportMessageReceivedMessage: String?
+	var reportMessageClosure: ((String) -> Void)?
 
     func report(message: String) {
         reportMessageCallsCount += 1
         reportMessageReceivedMessage = message
+		reportMessageClosure?(message)
     }
+
 }
 class InitializationProtocolMock: InitializationProtocol {
 
     //MARK: - init
 
     var initIntParameterStringParameterOptionalParameterReceivedArguments: (intParameter: Int, stringParameter: String, optionalParameter: String?)?
+	var initIntParameterStringParameterOptionalParameterClosure: ((Int, String, String?) -> Void)?
 
     required init(intParameter: Int, stringParameter: String, optionalParameter: String?) {
         initIntParameterStringParameterOptionalParameterReceivedArguments = (intParameter: intParameter, stringParameter: stringParameter, optionalParameter: optionalParameter)
+		initIntParameterStringParameterOptionalParameterClosure?(intParameter, stringParameter, optionalParameter)
     }
     //MARK: - start
 
     var startCallsCount = 0
     var startCalled: Bool {
         return startCallsCount > 0
     }
+	var startClosure: (() -> Void)?
 
     func start() {
         startCallsCount += 1
+		startClosure?()
```

Comment:
whitespaces feel off, can you adjust the template so they match other formatting?